README.md

```diff
@@ -183,6 +183,82 @@ Blank nodes are not supported.
   }
 }
 ```
+#### Resource Object Fragmentation Shape
+
+A fragmentation strategy that groups triples by (subject) resources.
+It generates shapes and shape index files in the root of the iri container for each information type defined in `shapeDirectory`. 
+The `shapeDirectory` must contain a `config.json` following the exemple below.
+The `shape` must be a path relative to the `shapeDirectory` and the `directory` must be the name of container of the subject.
+The shape located at `shape` must be of the format [`ShExC`](https://shex.io/shex-semantics/index.html#shexc).
+
+`./shape/config.json`
+```json
+{
+    "shapes": {
+        "comments": {
+            "shape": "comments.shexc",
+            "directory": "comments"
+        },
+    }
+}
+```
+
+An exemple of the `component.js` configuration is presented below.
+A probability for the generation of shape information of eachs resource can be defined with `generationProbability` in percentage.
+
+```json
+{
+  "fragmentationStrategy": { 
+        "@type": "FragmentationStrategyShape",
+        "shapeDirectory": "./shape",
+        "tripleShapeTreeLocator": true,
+        "generationProbability": 50
+      }
+}
+```
+A sample output file tree and its associated file is displayed below.
+
+```
+├── comments
+│   ├── 2012-08-10.nq
+│   └── 2012-08-17.nq
+├── comments_shape.nq
+├── noise
+│   ├── NOISE-12310.nq
+│   └── NOISE-9909.nq
+├── posts.nq
+├── posts_shape.nq
+├── profile
+│   └── card.nq
+├── profile_shape.nq
+├── settings
+│   └── publicTypeIndex.nq
+└── shapetree.nq
+```
+
+
+`http://localhost:3000/pods/00000000000000000768/profile_shape`
```

Comment:
What is the purpose of this profile shape? It doesn't seem to contain any actuale shape.

Reply:
I didn't wanted to wrote the whole output because it's kinda long so I just left those few triples.

Reply:
Could you add part of the output? Because the current output doesn't say a lot (I don't know what it will output).